projects/js-packages/publicize-components/src/components/social-previews/linkedin.js

```diff
@@ -13,7 +14,9 @@ import { getLinkedInDetails } from '../../store/selectors';
 export function LinkedIn( props ) {
 	const { title, url, image, media } = props;
 
-	const { name, profileImage } = getLinkedInDetails();
+	const { displayName: name, profileImage } = useSelect( select =>
+		select( SOCIAL_STORE_ID ).getConnectionProfileDetails( 'linkedin' )
```

Comment:
Should these connection strings be constants somewhere?

Reply:
That is a good suggestion, thanks.

Reply:
Done ✅